lib/challenge.js

```diff
@@ -52,6 +70,15 @@ var some_pet;
 
 // Answer Starts Here
 
+var some_pet = {
+
+  name: "Bailey Boxer",
+  species: "Boxer",
+  breed: "mixed",
+  noise: "Grrrr"
+  ],
```

Comment:
This loose, unmatched closing square bracket is a syntax error. Commas are also not needed following the final key-value pair in an object literal.